srcpkgs/rusty-diceware/patches/0001-Fix-values-in-test-cases.patch

```diff
@@ -0,0 +1,38 @@
+From 9f480fcbc9cb7bfb0d0bde8727dbfba9d878b596 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jan=20Christian=20Gr=C3=BCnhage?=
+ <jan.christian@gruenhage.xyz>
+Date: Sun, 4 Sep 2022 14:38:10 +0200
+Subject: [PATCH] Fix values in test cases
```

Comment:
Submitted upstream in https://gitlab.com/yuvallanger/rusty-diceware/-/merge_requests/2